modules/core/src/geojson-types.js

```diff
@@ -59,7 +65,7 @@ export type BoundingBoxArray = [number, number, number, number];
 export type FeatureOf<T: Geometry> = {
   type: 'Feature',
   geometry: T,
-  properties?: {},
+  properties?: { _internalIndex?: number },
```

Comment:
Do you still need `_internalIndex`?

Reply:
This is also no longer needed. I made sure that there are no more references to `_internalIndex`.